main.js

```diff
@@ -0,0 +1,38 @@
+const render = new Renderer()
+const apiManager = new APIManager()
+
+const $displaySavedUsers    = $('#display-saved-users')
+const $displaytUserData     = $('#display-user')
+const $getUserData          = $('#load-user')
+const $saveUser             = $('#save-user')
+
+
+$getUserData.on("click", function(){
+    apiManager.getPokemon()
+    apiManager.getPeople()
+    apiManager.getQuote()
+    apiManager.getInfo()
+})
```

Comment:
you could create a method in the apiManager which invokes all those inner methods